File: myonlinefm/src/main/java/com/sec/myonlinefm/xmlcheck/LocalInfoParser.java
package com.sec.myonlinefm.xmlcheck;

import java.io.InputStream;
import java.util.List;

/**
 * Created by SRC-TJ-MM-BinYang on 2018/1/3.
 */

public interface LocalInfoParser {
    public List<LocalInfo> parse(InputStream is) throws Exception;
    public String serialize(List<LocalInfo> books) throws Exception;
}

File: myonlinefm/src/main/java/com/sec/myonlinefm/xmlcheck/LocalInfoParser.kt
package com.sec.myonlinefm.xmlcheck

import java.io.InputStream

/**
 * Created by SRC-TJ-MM-BinYang on 2018/1/3.
 */
interface LocalInfoParser {
    @Throws(Exception::class)
    open fun parse(`is`: InputStream?): MutableList<LocalInfo?>?
    @Throws(Exception::class)
    open fun serialize(books: MutableList<LocalInfo?>?): String?
}